src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1162,14 +1164,34 @@ protected static void VerifyFlowGraphForTest<TSyntaxNode>(CSharpCompilation comp
                 stringBuilder.Builder.AppendLine($"    Statements ({statements.Length})");
                 foreach (var statement in statements)
                 {
+                    ValidateRoot(statement);
                     stringBuilder.Builder.AppendLine(OperationTreeVerifier.GetOperationTree(compilation, statement, initialIndent: 8));
                 }
 
                 if (block.Conditional.Condition != null)
                 {
                     Assert.True(map.TryGetValue(block.Conditional.Destination, out int index));
                     stringBuilder.Builder.AppendLine($"    Jump if {block.Conditional.JumpIfTrue} to Block[{index}]");
-                    stringBuilder.Builder.AppendLine(OperationTreeVerifier.GetOperationTree(compilation, block.Conditional.Condition, initialIndent: 8));
+
+                    IOperation condition = block.Conditional.Condition;
+                    ValidateRoot(condition);
+                    stringBuilder.Builder.AppendLine(OperationTreeVerifier.GetOperationTree(compilation, condition, initialIndent: 8));
+                }
+
+                void ValidateRoot(IOperation root)
```

Comment:
Please use naming convention for locals (camelCase). It helps readability.

Reply:
> Please use naming convention for locals (camelCase). It helps readability.

I'll address this in the next PR, if that is Ok.

---
In reply to: [163069024](https://github.com/dotnet/roslyn/pull/24358#discussion_r163069024) [](ancestors = 163069024)

Reply:
We should consider using uppercase for first letter of local functions. (It shouldn't matter to the caller whether the function is defined in the method or outside.)

Reply:
It matters, as methods don't capture.